### PR TITLE
INTLY-4100 standardise on pds cluster type

### PR DESCRIPTION
**Pull request by aidenkeating**
verification:
- run the installer using the pds inventory file
- ensure nexus is created

### CODE REVIEW

File: inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 integreatly_version: master
 
-# Possible values are: rhpds, poc, osd, dev
-cluster_type: rhpds
+# Possible values are: pds, poc, osd, dev
+cluster_type: pds
```
Comment (pmccarthy) [inline inventories/group_vars/all/manifest.yaml]: Given that we set `cluster_type` in each of the environment specific group_vars we can probably just remove this altogether or at the very least, set as an empty string

**Comment by david-martin** [comment]
/lgtm
/approve

@aidenkeating to follow up with the removal of hosts.template and some docs to remove any confusion on how to run the installer after recent inventory changes

**Comment by aidenkeating** [comment]
@david-martin Thanks for taking a look, follow-up tasks captured in https://issues.jboss.org/browse/INTLY-4312

File: inventories/group_vars/all/manifest.yaml (hunk 1 above)
Comment (jessesarn) [reply inventories/group_vars/all/manifest.yaml]: Seems removing altogether may have had a side effect of breaking the integreatly installer on RHPDS (https://issues.redhat.com/browse/INTLY-4368)

Without a value being set by default, cluster_type is left undefined which none of the conditionals which are based on the value of the cluster_type handle gracefully.  Setting the value to rhpds may not have triggered any special logic, but it did ensure the variable was defined.

I haven't had enough time to look through all of the logic where something pds specific needs to happen to feel comfortable re-adding the property with a blank string and expecting it to use the pds commons/template.  For now, I'm going to try installing 1.6.0-rhpds with the value defaulted to pds to unblock the 1.6/pds testing.

**Pull request merged**